### ISSUE TITLE
来贡献模型了，aishell3的

### ISSUE THREAD

**Issue by ferretgeek** [babysor/MockingBird#165]
https://pan.baidu.com/s/1pu_XfQJnLRcQZYfawqCeNQ ，提取码：7777

aishell3数据集，Tesla V100 32G，BS 96训练的160K，loss值0.24

同时本人有两台V100 32G闲置，为BUG时撸的云服务器，有想训练啥的也可以提要求，反正闲着也是闲着。

**Comment by cocucola** [comment]
这个模型效果不错。

**Comment by ZHAOYANZHOU** [comment]
> > https://pan.baidu.com/s/1pu_XfQJnLRcQZYfawqCeNQ ，提取码：7777
> > aishell3数据集，Tesla V100 32G，BS 96训练的160K，loss值0.24
> > 同时本人有两台V100 32G闲置，为BUG时撸的云服务器，有想训练啥的也可以提要求，反正闲着也是闲着。
> 
> 新版本兼容性有问题，等待作者修复，现在要用这个模型请用我分享的旧版本。 链接：https://pan.baidu.com/s/14UweWwENPc0myDLezy8L3Q 提取码：7777

这个链接失效了，可以劳烦再发一次嘛

**Comment by ferretgeek** [comment]
> > > https://pan.baidu.com/s/1pu_XfQJnLRcQZYfawqCeNQ ，提取码：7777
> > > aishell3数据集，Tesla V100 32G，BS 96训练的160K，loss值0.24
> > > 同时本人有两台V100 32G闲置，为BUG时撸的云服务器，有想训练啥的也可以提要求，反正闲着也是闲着。
> > 
> > 
> > 新版本兼容性有问题，等待作者修复，现在要用这个模型请用我分享的旧版本。 链接：https://pan.baidu.com/s/14UweWwENPc0myDLezy8L3Q 提取码：7777
> 
> 这个链接失效了，可以劳烦再发一次嘛

无需使用这个了，在本项目主页的tag当中，有作者发布的旧版本压缩包

**Comment by ZHAOYANZHOU** [comment]
> > > > https://pan.baidu.com/s/1pu_XfQJnLRcQZYfawqCeNQ ，提取码：7777
> > > > aishell3数据集，Tesla V100 32G，BS 96训练的160K，loss值0.24
> > > > 同时本人有两台V100 32G闲置，为BUG时撸的云服务器，有想训练啥的也可以提要求，反正闲着也是闲着。
> > > 
> > > 
> > > 新版本兼容性有问题，等待作者修复，现在要用这个模型请用我分享的旧版本。 链接：https://pan.baidu.com/s/14UweWwENPc0myDLezy8L3Q 提取码：7777
> > 
> > 
> > 这个链接失效了，可以劳烦再发一次嘛
> 
> 无需使用这个了，在本项目主页的tag当中，有作者发布的旧版本压缩包

好的，非常感谢！

> > > > https://pan.baidu.com/s/1pu_XfQJnLRcQZYfawqCeNQ ，提取码：7777
> > > > aishell3数据集，Tesla V100 32G，BS 96训练的160K，loss值0.24
> > > > 同时本人有两台V100 32G闲置，为BUG时撸的云服务器，有想训练啥的也可以提要求，反正闲着也是闲着。
> > > 
> > > 
> > > 新版本兼容性有问题，等待作者修复，现在要用这个模型请用我分享的旧版本。 链接：https://pan.baidu.com/s/14UweWwENPc0myDLezy8L3Q 提取码：7777
> > 
> > 
> > 这个链接失效了，可以劳烦再发一次嘛
> 
> 无需使用这个了，在本项目主页的tag当中，有作者发布的旧版本压缩包

好的，非常感谢！

**Comment by joshua54321** [comment]
这个模型我载入测试时，效果还是不好，目前我测试下来效果最好的还是作者发布的那个ceshi.pt，请问是有哪些地方配置不对吗？（另外，其实也很想知道作者训练ceshi.pt时的参数配置，如何训练重现，因为我训练的几个模型效果也是很差很差）

**Comment by ferretgeek** [comment]
> 

这个模型实际上我未曾测试过，只是看着loss好看发出来。我建议你可以测试我另外一个aishell3 160K的，那个我有实测过效果不错。

**Comment by ferretgeek** [comment]
> 这个模型我载入测试时，效果还是不好，目前我测试下来效果最好的还是作者发布的那个ceshi.pt，请问是有哪些地方配置不对吗？（另外，其实也很想知道作者训练ceshi.pt时的参数配置，如何训练重现，因为我训练的几个模型效果也是很差很差）

这个模型实际上我未曾测试过，只是看着loss好看发出来。我建议你可以测试我另外一个aishell3 160K的，那个我有实测过效果不错。

**Comment by joshua54321** [comment]
> 



> > 这个模型我载入测试时，效果还是不好，目前我测试下来效果最好的还是作者发布的那个ceshi.pt，请问是有哪些地方配置不对吗？（另外，其实也很想知道作者训练ceshi.pt时的参数配置，如何训练重现，因为我训练的几个模型效果也是很差很差）
> 
> 这个模型实际上我未曾测试过，只是看着loss好看发出来。我建议你可以测试我另外一个aishell3 160K的，那个我有实测过效果不错。

我就是下载的您云盘分享的“aishell3  160K  BS96模型”，这个是您说的实测过的吧？

**Comment by ferretgeek** [comment]
> 

这个在真人语音，只用9句话的情况下，跑40K即可有非常不错的效果。另外在游戏角色的语音克隆下也有一点勉强可用的效果。不知道你的自定义数据集多大，也许你用少量的自定义数据集跑一下会有不错的效果

**Comment by ferretgeek** [comment]
> > 
> 
> > > 这个模型我载入测试时，效果还是不好，目前我测试下来效果最好的还是作者发布的那个ceshi.pt，请问是有哪些地方配置不对吗？（另外，其实也很想知道作者训练ceshi.pt时的参数配置，如何训练重现，因为我训练的几个模型效果也是很差很差）
> > 
> > 
> > 这个模型实际上我未曾测试过，只是看着loss好看发出来。我建议你可以测试我另外一个aishell3 160K的，那个我有实测过效果不错。
> 
> 我就是下载的您云盘分享的“aishell3 160K BS96模型”，这个是您说的实测过的吧？

这个在真人语音，只用9句话的情况下，跑40K即可有非常不错的效果。另外在游戏角色的语音克隆下也有一点勉强可用的效果。不知道你的自定义数据集多大，也许你用少量的自定义数据集跑一下会有不错的效果

**Comment by joshua54321** [comment]
> > > 
> > 
> > 
> > > > 这个模型我载入测试时，效果还是不好，目前我测试下来效果最好的还是作者发布的那个ceshi.pt，请问是有哪些地方配置不对吗？（另外，其实也很想知道作者训练ceshi.pt时的参数配置，如何训练重现，因为我训练的几个模型效果也是很差很差）
> > > 
> > > 
> > > 这个模型实际上我未曾测试过，只是看着loss好看发出来。我建议你可以测试我另外一个aishell3 160K的，那个我有实测过效果不错。
> > 
> > 
> > 我就是下载的您云盘分享的“aishell3 160K BS96模型”，这个是您说的实测过的吧？
> 
> 这个在真人语音，只用9句话的情况下，跑40K即可有非常不错的效果。另外在游戏角色的语音克隆下也有一点勉强可用的效果。不知道你的自定义数据集多大，也许你用少量的自定义数据集跑一下会有不错的效果

感谢您的回复，我后来基于最新的代码又测了一下您的模型，是可以work的。同时，也想和您交流一下，请教一下您，我现在训练synthesizor模型，用aidatatang200Zh语料，batch_size为30（GPU的显存有限，只能用这么大的size了），loss基本就在0.37左右徘徊无法下降了。也正在其他的数据集，但我有个疑问，先不考虑其他因素，但从数据量来看，200h的数据loss无法下降，数据量的增大会不会对效果的影响也会有限呢。。。

**Comment by ferretgeek** [comment]
> > > > 
> > > 
> > > 
> > > > > 这个模型我载入测试时，效果还是不好，目前我测试下来效果最好的还是作者发布的那个ceshi.pt，请问是有哪些地方配置不对吗？（另外，其实也很想知道作者训练ceshi.pt时的参数配置，如何训练重现，因为我训练的几个模型效果也是很差很差）
> > > > 
> > > > 
> > > > 这个模型实际上我未曾测试过，只是看着loss好看发出来。我建议你可以测试我另外一个aishell3 160K的，那个我有实测过效果不错。
> > > 
> > > 
> > > 我就是下载的您云盘分享的“aishell3 160K BS96模型”，这个是您说的实测过的吧？
> > 
> > 
> > 这个在真人语音，只用9句话的情况下，跑40K即可有非常不错的效果。另外在游戏角色的语音克隆下也有一点勉强可用的效果。不知道你的自定义数据集多大，也许你用少量的自定义数据集跑一下会有不错的效果
> 
> 感谢您的回复，我后来基于最新的代码又测了一下您的模型，是可以work的。同时，也想和您交流一下，请教一下您，我现在训练synthesizor模型，用aidatatang200Zh语料，batch_size为30（GPU的显存有限，只能用这么大的size了），loss基本就在0.37左右徘徊无法下降了。也正在其他的数据集，但我有个疑问，先不考虑其他因素，但从数据量来看，200h的数据loss无法下降，数据量的增大会不会对效果的影响也会有限呢。。。

作者本人倾向于，多数据集的混合训练，以及微调训练参数。所以如果你loss下不去，就考虑调整学习率（learning rate），亦或者换数据集、增加数据集这种手段来解决。至于效果到底如何，还是需要你去实测才知道。

因为不同batch size的大小也能影响效果，我在训练数据集时发现，aidatatang200_zh混合aishell3，loss值的下降速度比单纯训练1个数据集，要快得多。但是loss也不代表最终效果，你可以看到我分享的这个70K虽然loss很好看，但据反馈效果不行，所以我个人觉得混合数据集也需要增加训练步数才会有更好的效果

--以上基于本人纯猜想及经验，具体以实践为真。

**Comment by joshua54321** [comment]
> 合数据集也需要增加训练步数才会有更好的效果

好的，非常感谢您的回复和经验分享，我最近也在尝试，同时后面时间允许也会认真看下代码，期待与您做进一步交流，感谢！

**Comment by xiuxiu666666** [comment]
> https://pan.baidu.com/s/1pu_XfQJnLRcQZYfawqCeNQ ，提取码：7777
> 
> aishell3数据集，Tesla V100 32G，BS 96训练的160K，loss值0.24
> 
> 同时本人有两台V100 32G闲置，为BUG时撸的云服务器，有想训练啥的也可以提要求，反正闲着也是闲着。

您好，我用您这个模型，报这样的错误，我该怎么办
Caught exception: RuntimeError('Error(s) in loading state_dict for Tacotron:\n\tsize mismatch for encoder_proj.weight: copying a param with shape torch.Size([128, 512]) from checkpoint, the shape in current model is torch.Size([128, 1024]).\n\tsize mismatch for gst.stl.attention.W_query.weight: copying a param with shape torch.Size([512, 256]) from checkpoint, the shape in current model is torch.Size([512, 512]).\n\tsize mismatch for decoder.attn_rnn.weight_ih: copying a param with shape torch.Size([384, 768]) from checkpoint, the shape in current model is torch.Size([384, 1280]).\n\tsize mismatch for decoder.rnn_input.weight: copying a param with shape torch.Size([1024, 640]) from checkpoint, the shape in current model is torch.Size([1024, 1152]).\n\tsize mismatch for decoder.stop_proj.weight: copying a param with shape torch.Size([1, 1536]) from checkpoint, the shape in current model is torch.Size([1, 2048]).',)
Restarting

**Comment by 1nlplearner** [comment]
您好，请问训练的时候有遇到过训练过一段时间后，loss跑飞了的情况吗

**Comment by alivic505** [comment]
ارحب
